scripts/webpack/webpack.base.dev.js

```diff
@@ -85,9 +85,25 @@ module.exports = {
             },
           },
           {
-            loader: 'ts-loader',
-            options: { transpileOnly: true },
+            loader: 'babel-loader',
```

Comment:
Add `babel-loader` to webpack dev so that we can start development site on IE11 because IE11 cannot understand ES6 syntax